cinema/models.py

```diff
@@ -84,31 +88,40 @@ class Ticket(models.Model):
     row = models.IntegerField()
     seat = models.IntegerField()
 
-    def clean(self):
-        for ticket_attr_value, ticket_attr_name, cinema_hall_attr_name in [
-            (self.row, "row", "rows"),
-            (self.seat, "seat", "seats_in_row"),
-        ]:
-            count_attrs = getattr(
-                self.movie_session.cinema_hall, cinema_hall_attr_name
-            )
-            if not (1 <= ticket_attr_value <= count_attrs):
-                raise ValidationError(
-                    {
-                        ticket_attr_name: f"{ticket_attr_name} "
-                        f"number must be in available range: "
-                        f"(1, {cinema_hall_attr_name}): "
-                        f"(1, {count_attrs})"
-                    }
-                )
+    @staticmethod
+    def validate_seat_and_row(
+            param: str,
+            number: int,
+            max_number: int,
+            error_to_raise: error
+    ) -> Optional[error]:
```

Comment:
It is not fixed. The function returns nothing - `raise error != return`
So the annotation should be None